python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java

```diff
@@ -248,6 +248,41 @@ public static List<FunctionDef> topLevelFunctionDefs(ClassDef classDef) {
     return visitor.functionDefs;
   }
 
+  public static int findIndent(Tree tree) {
```

Comment:
I would suggest to rename it to `findIndentationSize`

Reply:
done